init.js

```diff
@@ -42,11 +42,11 @@ db.sequelize.sync({ force: true })
     return db.Role.create({ name: 'public' });
   })
   .then(function () {
-    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/'})
+    return db.Page.create({title: 'Welcome!', content: 'Welcome to NoBS.js!', slug:'/index'});
```

Comment:
Need a name of some sort for the state. It used to be 'pages.1'. We're now generating the state names from the slugs as opposed to from the page id values. So the slug '/' is converted to '' which cannot be a state name. 

I could add some logic to the page creation function that checks for 'index' as the state name and uses the url '/' instead of '/index' but I feel that would be a separate issue at this point.